lib/jsdom/living/nodes/GlobalEventHandlers.webidl

```diff
@@ -1,4 +1,4 @@
-// https://html.spec.whatwg.org/#globaleventhandlers
+// https://html.spec.whatwg.org/multipage/webappapis.html#globaleventhandlers
```

Comment:
Please don't make unrelated changes in the PR. You can file issues asking if they are OK but it is not good to do so unsolicited.